jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/PluginResource.java

```diff
@@ -98,7 +97,6 @@ public PluginResource(@Named("osgi") final HttpServlet osgiServlet, // See Defau
         this.osgiServlet = osgiServlet;
     }
 
-    @Timed
     @DELETE
```

Comment:
@sbrossie and I talked about this. We ended up deciding to remove the metric generation from this resource since they don't provide that much info.
Here's what the metric name would look like: `"kb_resource./plugins{subResources:.\*}.doPUT.PUT.2xx.230"`

Since plugins register servlets through OSGI to serve these requests, and we have metric for those, we'd be covered.